lib/matplotlib/axes/_axes.py

```diff
@@ -3194,6 +3194,9 @@ def stem(self, *args, linefmt=None, markerfmt=None, basefmt=None, bottom=0,
         baseline, = self.plot(baseline_x, baseline_y,
                               color=basecolor, linestyle=basestyle,
                               marker=basemarker, label="_nolegend_")
+        if self.name == "polar" and orientation == 'vertical':
```

Comment:
I'm not sure it's necessary to check the Axes type, as interpolation may be useful in e.g., Cartopy (though I'm not so sure `stem` makes sense there, I just mean other projections), and the other users don't gate on it either.

Reply:
Indeed, it seems this is a hinting attribute for transforms. From our builtin transforms only PolarTransform is using it. Thus, we need no gating - other transforms simply ignore it.